components/base_page.js

```diff
@@ -11,10 +12,23 @@ import css from '../styles/common.module.scss';
 import utilsCss from '../styles/utils.module.scss';
 
 export default function Page({ children }) {
+    const router = useRouter();
+    const path = router.pathname;
+    console.log(path);
     return (
         <div className={css.container}>
             <Head>
-                <title>Awe and Reverence</title>
+                <title>
+                    {path
+                        .replace(/\//, '')
+                        .replace(
+                            /\w+/,
+                            path.charAt(1).toUpperCase() +
+                                path.substring(2, path.length) +
+                                ' | '
+                        )}
+                    Awe & Reverence
```

Comment:
Can we define the string `Awe & Reverence` as constant `const SITE_TITLE = 'Awe & Reverence';` somewhere, like in a file like `components/constants.js`, and just use the constant everywhere?

Reply:
I created a file called components/constants.js and implemented it wherever necessary.